aws-base-1/terragrunt.hcl

```diff
@@ -3,7 +3,7 @@ include "root" {
 }
 
 terraform {
-  source = "github.com/mjmenger/terraform-f5xc-aws-base.git?ref=v0.0.1"
+  source = "github.com/mjmenger/terraform-f5xc-aws-base.git?ref=natgw"
```

Comment:
this should be changed to a numbered release before merging into main

Reply:
Agreed, I wanted you to review the source module first since you're the owner. I have a PR on it. Ideally it is ref=v0.0.2 for example. But again, I wanted you to review PR and tag/rev yourself. See below PR.

https://github.com/mjmenger/terraform-f5xc-aws-base/pull/4

Reply:
From our discussion, i will tag my branch and update to tag version